nupic/research/frameworks/vernon/mixins/oml.py

```diff
@@ -152,7 +216,7 @@ def find_best_lr(self, num_classes_learned):
         lr_all = []
 
         # Grid search over lr
-        for _lr_search_runs in range(0, self.num_lr_search_runs):
+        for _lr_search_run in range(0, self.num_lr_search_runs):
```

Comment:
The _ is to signal this variable is not being used? I would rather use just a regular underscore, I found myself looking for it to understand the meaning of this variable.

Reply:
Changed it